tests/test_config.py

```diff
@@ -2388,6 +2400,172 @@ def test_model_type_llm(self):
             config.inference_output_fields, DEFAULT_LLM_INFERENCE_OUTPUT_FIELDS
         )
 
+    def test_config_parameter_creation_default(self):
+        """
+        Test that config parameters are correctly created in default optuna case
+        """
+
+        args = [
+            "model-analyzer",
+            "profile",
+            "--model-repository",
+            "cli-repository",
+            "-f",
+            "path-to-config-file",
+            "--run-config-search-mode",
+            "optuna",
+        ]
+
+        yaml_content = """
+        profile_models: add_sub
+        """
+
+        config = self._evaluate_config(args, yaml_content)
+
+        analyzer = Analyzer(config, MagicMock(), MagicMock(), MagicMock())
+        analyzer._populate_search_parameters()
+
+        batch_sizes = analyzer._search_parameters["add_sub"].get_parameter(
+            "batch_sizes"
+        )
+        self.assertEqual(ParameterType.MODEL, batch_sizes.ptype)
```

Comment:
Could you add comment headers to visually organize the test (helps with skimming for understanding and finding the parts you need)? E.g. this section is for batch sizes, next one is for concurrency sizes, etc.